### PR TITLE
Added travis ci support

### PR DESCRIPTION
**Pull request by amilajack**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+node_js:
+  - 6
+  - 4
+
+cache:
+  directories:
+    - node_modules
+
+install:
```
Comment (vigneshshanmugam) [inline .travis.yml:13]: you don't need install here. modules will be installed before running script.

**Review by vigneshshanmugam — COMMENTED** [review]
Thanks for the PR @amilajack :)

File: .travis.yml (hunk 1 above)
Comment (amilajack) [reply .travis.yml:13]: This line forces the installation to happen with the latest npm version. It overrides the default

Comment (vigneshshanmugam) [reply .travis.yml:13]: Alright 👍

**Review by vigneshshanmugam — APPROVED** [review]

**Pull request merged**